src/test/java/com/epam/eco/commons/avro/modification/SortSchemaFieldsTest.java

```diff
@@ -123,3 +124,22 @@ private void verifyFieldsOrdered(List<Map<String, Object>> fields, boolean asc)
     }
 
 }
+
+class MockByNameSchemaFieldComparator implements Comparator<Map<String, Object>> {
```

Comment:
Not sure if MockXxx is a good name... Let's better name one as SimpleByNameSchemaFieldComparator